cli/telemetry/telemetry.go

```diff
@@ -62,6 +63,9 @@ func ReportAPICalls(apiProvider *apiserver.Provider) {
 }
 
 func shouldRunTelemetry() bool {
+	if _, present := os.LookupEnv("MIZU_DISABLE_TELEMTRY"); present {
```

Comment:
Maybe change to const

Reply:
I can do it, but can't see any advantage to it